qwiic_i2c/circuitpython_i2c.py

```diff
@@ -330,7 +330,14 @@ def isDeviceConnected(self, devAddress):
 			self._i2cbus.writeto(devAddress, bytearray())
 			isConnected = True
 		except:
```

Comment:
This catches all exceptions, even for non-ESP32 platforms where the device is not connected, so it would test twice on those platforms. Not necessarily a problem, but just wondering if we should go about this a different way to ensure the test only happens once? Idk, probably not a big deal.

Reply:
Fair points! Ship it!